### ISSUE TITLE
Run code in specific versions

### ISSUE THREAD

**Issue by NeonSupernova** [mobibean/codesnack-ide-feature-requests#84]
So, I would like different container versions than what is typically there, namely Python. I would like Python 3.10, but importing custom docker containers doesn't seem to be available. It would be nice to have a drop-down menu here:
![image](https://user-images.githubusercontent.com/78397254/139555521-627c9316-444a-4afd-8a2f-1e15dd70c8ad.jpeg)
Because with all the different java versions it looks kinda messy. Anyway, “latest” does not specify what version it is, which can matter when programs get complex. 

## TL/DR
Add dropdown menu for container versions.
Add more versions for at least python.
Show version numbers instead of “latest”

**Comment by bushev** [comment]
Hey @NeonSupernova,

We are rethinking how the containers work. This selection will be replaced with just one container image that will contain all the languages plus an option to customize the installation of extra compilers.

To solve your current issue, I've added Python 3.10 tag to that list. Please, check it out in the app.

| ![photo_2021-11-03_09-45-27](https://user-images.githubusercontent.com/7535766/140071521-919ea95f-e0a3-4446-89ae-2165f3d3898e.jpg)  | ![photo_2021-11-03_09-45-26](https://user-images.githubusercontent.com/7535766/140071567-0aaf4924-0914-41d6-9727-ef2fd24d8fb7.jpg) |
| ------------- | ------------- |
